testsuite/MDAnalysisTests/coordinates/test_lammps.py

```diff
@@ -393,3 +395,81 @@ def test_write_non_numerical_types(self, tmpout):
             assert 'must be convertible to integers' in e.args[0]
         else:
             raise pytest.fail()
+
+
+class TestLammpsDumpReader(object):
+    @pytest.fixture()
```

Comment:
You could save your data file as bz2 compressed and then create fixtures that create the uncompressed, gz, and zip versions for the test.